packages/cache/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "4.0.5",
   "preview": true,
   "description": "Actions cache lib",
+  "engines": {
+    "node": ">=24.0.0"
+  },
```

Comment:
This will cause any package that depends on `@actions/cache` to also require Node 24, removing backward compatibility.

Reply:
The plan is to have a major version bump here as well as in actions cache anyway, so version of cache that already references the current or older versions won't be referencing this change, and any newer versions would be on the action that targets node24. What scenario do you envision needing a runner that doesn't have node 24? Self-hosted runners should all be updated and all hosted runners have node 24 already.